Source/SwiftLintCore/Extensions/Configuration+FileGraph.swift

```diff
@@ -88,18 +88,42 @@ package extension Configuration {
             )
 
             if !ignoreParentAndChildConfigs {
-                try processPossibleReference(
+                try processPossibleReferenceIgnoringFileAbsence(
                     ofType: .childConfig,
                     from: vertex,
                     remoteConfigTimeoutOverride: remoteConfigTimeoutOverride,
-                    remoteConfigTimeoutIfCachedOverride: remoteConfigTimeoutIfCachedOverride
-                )
-                try processPossibleReference(
+                    remoteConfigTimeoutIfCachedOverride: remoteConfigTimeoutIfCachedOverride)
+
+                try processPossibleReferenceIgnoringFileAbsence(
                     ofType: .parentConfig,
                     from: vertex,
                     remoteConfigTimeoutOverride: remoteConfigTimeoutOverride,
+                    remoteConfigTimeoutIfCachedOverride: remoteConfigTimeoutIfCachedOverride)
+            }
+        }
+
+        private mutating func processPossibleReferenceIgnoringFileAbsence(
+            ofType type: EdgeType,
+            from vertex: Vertex,
+            remoteConfigTimeoutOverride: TimeInterval?,
+            remoteConfigTimeoutIfCachedOverride: TimeInterval?
+        ) throws {
+            do {
+                try processPossibleReference(
+                    ofType: type,
+                    from: vertex,
+                    remoteConfigTimeoutOverride: remoteConfigTimeoutOverride,
                     remoteConfigTimeoutIfCachedOverride: remoteConfigTimeoutIfCachedOverride
                 )
+            } catch {
+                // If a child or parent config file doesn't exist, do not fail the rest of the config tree
+                // Instead just ignore this leaf of the config
+                // Otherwise, rethrow the error.
```

Comment:
```suggestion
                // If a child or parent config file doesn't exist, do not fail the rest of the config tree.
                // Instead, just ignore this leaf of the config. Otherwise, rethrow the error.
```